uniro_frontend/src/pages/buildingSearch.tsx

```diff
@@ -25,12 +27,19 @@ export default function BuildingSearchPage() {
 			}),
 	},)
 
+	const handleBack = () => {
+		navigate(-1);
```

Comment:
unversity가 저장되어 있는 상태에서 바로 접근이 가능한 페이지라,
```suggestion
		navigate("/map");
```
으로 넘어가는 방식도 괜찮을 것 같습니다!